### ISSUE TITLE
HydraulicModules fail in OpenModelica

### ISSUE THREAD

**Issue by FWuellhorst** [RWTH-EBC/AixLib#1346]
**Describe the bug**
The models in the package `HydraulicModules` fail in OpenModelica.

**Error log**

```
 Error in model:  AixLib.Systems.HydraulicModules.BaseClasses.BasicPumpInterface 
[D:/05_Programme/OpenModelica1.19.2-64bit/lib/omlibrary/Modelica 4.0.0/Media/package.mo:4426:7-4429:18:writable] Error: Called function ‘AixLib.Systems.HydraulicModules.BaseClasses.BasicPumpInterface.Medium.specificEnthalpy‘ is partial.


 
 Error in model:  AixLib.Systems.HydraulicModules.BaseClasses.PartialHydraulicModule 
[D:/05_Programme/OpenModelica1.19.2-64bit/lib/omlibrary/Modelica 4.0.0/Media/package.mo:4426:7-4429:18:writable] Error: Called function ‘AixLib.Systems.HydraulicModules.BaseClasses.PartialHydraulicModule.Medium1.specificEnthalpy‘ is partial.


 
 Error in model:  AixLib.Systems.HydraulicModules.BaseClasses.PumpInterface_PumpHeadControlled 
[D:/05_Programme/OpenModelica1.19.2-64bit/lib/omlibrary/Modelica 4.0.0/Media/package.mo:4109:7-4109:67:writable] Error: Output parameter state was not assigned a value


 
 Error in model:  AixLib.Systems.HydraulicModules.BaseClasses.PumpInterface_PumpSpeedControlled 
[D:/05_Programme/OpenModelica1.19.2-64bit/lib/omlibrary/Modelica 4.0.0/Media/package.mo:4109:7-4109:67:writable] Error: Output parameter state was not assigned a value


 
 Error in model:  AixLib.Systems.HydraulicModules.BaseClasses.PumpInterface_SpeedControlledNrpm 
[D:/05_Programme/OpenModelica1.19.2-64bit/lib/omlibrary/Modelica 4.0.0/Media/package.mo:4109:7-4109:67:writable] Error: Output parameter state was not assigned a value


 
 Error in model:  AixLib.Systems.HydraulicModules.Injection 
Error: Internal error Instantiation of AixLib.Systems.HydraulicModules.Injection failed with no error message.


 
 Error in model:  AixLib.Systems.HydraulicModules.Injection2WayValve 
[D:/05_Programme/OpenModelica1.19.2-64bit/lib/omlibrary/Modelica 4.0.0/Media/package.mo:4109:7-4109:67:writable] Error: Output parameter state was not assigned a value


 
 Error in model:  AixLib.Systems.HydraulicModules.Pump 
[D:/05_Programme/OpenModelica1.19.2-64bit/lib/omlibrary/Modelica 4.0.0/Media/package.mo:4109:7-4109:67:writable] Error: Output parameter state was not assigned a value


 
 Error in model:  AixLib.Systems.HydraulicModules.SimpleConsumer 
[D:/05_Programme/OpenModelica1.19.2-64bit/lib/omlibrary/Modelica 4.0.0/Media/package.mo:4109:7-4109:67:writable] Error: Output parameter state was not assigned a value


 
 Error in model:  AixLib.Systems.HydraulicModules.Throttle 
[D:/05_Programme/OpenModelica1.19.2-64bit/lib/omlibrary/Modelica 4.0.0/Media/package.mo:4109:7-4109:67:writable] Error: Output parameter state was not assigned a value


 
 Error in model:  AixLib.Systems.HydraulicModules.ThrottlePump 
[D:/05_Programme/OpenModelica1.19.2-64bit/lib/omlibrary/Modelica 4.0.0/Media/package.mo:4109:7-4109:67:writable] Error: Output parameter state was not assigned a value

 Error in model:  AixLib.Systems.HydraulicModules.Example.Admix 
Failed to build model: AixLib.Systems.HydraulicModules.Example.Admix
Error: Too few equations, under-determined system. The model has 2082 equation(s) and 2083 variable(s).


 
 Error in model:  AixLib.Systems.HydraulicModules.Example.Injection 
Failed to build model: AixLib.Systems.HydraulicModules.Example.Injection
Error: Too few equations, under-determined system. The model has 3017 equation(s) and 3018 variable(s).


 
 Error in model:  AixLib.Systems.HydraulicModules.Example.Injection2WayValve 
Failed to build model: AixLib.Systems.HydraulicModules.Example.Injection2WayValve
Error: Too few equations, under-determined system. The model has 2310 equation(s) and 2311 variable(s).


 
 Error in model:  AixLib.Systems.HydraulicModules.Example.Pump 
Failed to build model: AixLib.Systems.HydraulicModules.Example.Pump
Error: Too few equations, under-determined system. The model has 910 equation(s) and 911 variable(s).


 
 Error in model:  AixLib.Systems.HydraulicModules.Example.ThrottlePump 
Failed to build model: AixLib.Systems.HydraulicModules.Example.ThrottlePump
Error: Too few equations, under-determined system. The model has 1390 equation(s) and 1391 variable(s).
```

**Comment by FWuellhorst** [comment]
The TABS model fails with the same error message. If a solution is found, we can fix TABS as well:
```
 Error in model:  AixLib.Systems.TABS.Tabs 
[D:/05_Programme/OpenModelica1.19.2-64bit/lib/omlibrary/Modelica 4.0.0/Media/package.mo:4109:7-4109:67:writable] Error: Output parameter state was not assigned a value
```

**Comment by KremerMartin** [comment]
closed by PR #1359
